cumulus/parachains/integration-tests/emulated/common/src/lib.rs

```diff
@@ -62,6 +64,8 @@ pub const ASSET_HUB_ROCOCO_ID: u32 = 1000;
 pub const ASSET_HUB_WESTEND_ID: u32 = 1000;
 pub const ASSETS_PALLET_ID: u8 = 50;
 
+pub const WETH: [u8; 20] = hex!("fff9976782d46cc05630d1f6ebab18b2324d6b14");
```

Comment:
What is this `fff9976782d46cc05630d1f6ebab18b2324d6b14` value? at least at some description please.
Do we need this value here? Because this module is reused on the polkadot-fellows repo, so this `WETH` value will work there also? If it is just westend/rococo-eth-testnet related, I would prefer to remove it from here and define where needed - e.g. `bridge-hub-westend-emulated-chain`

Reply:
Added a description in https://github.com/paritytech/polkadot-sdk/pull/7402/commits/9bd919c469dee2c8aa2587674848f98c6acbf60f

Reply:
Tracking the larger issue in https://github.com/paritytech/polkadot-sdk/issues/7977